### ISSUE TITLE
Tuple name simplification botches formatting

### ISSUE THREAD

**Issue by rchande** [dotnet/roslyn#22347]
With language version 7.1, use the "simplify tuple name" feature to simplify the "a" member below:
```C#
            int a = 1;
            var t = (a: a, 2);
```
Note that the tuple is not reformatted afterwards leading to an extra space between the open paren and "a":
```C#
            int a = 1;
            var t = ( a, 2);
```

**Comment by dpoeschl** [comment]
FYI @jcouv -- I thought formatting was working here? Do you have pending work here (/do you want to take it), or should the IDE team take this?

**Comment by jcouv** [comment]
I'm happy to take it. My guess is that the trivia between `a:` and `a` is carried over.
My understanding of the general refactoring logic is that we do a formatting pass on rewritten nodes, is that right?

**Comment by dpoeschl** [comment]
Also at https://developercommunity.visualstudio.com/content/problem/122795/c-prefer-inferred-tuple-element-names-code-fix-lea.html

**Comment by dpoeschl** [comment]
Another: https://developercommunity.visualstudio.com/content/problem/122910/member-name-can-be-simplified-doesnt-apply-the-rig.html

**Comment by jcouv** [comment]
Fix merged (should release in 15.6).
